examples/closure-app/silabs/include/ClosureManager.h

```diff
@@ -29,10 +29,20 @@
 #include "ClosureControlEndpoint.h"
 #include "ClosureDimensionEndpoint.h"
 #include <lib/core/DataModelTypes.h>
+#include <AppEvent.h>
 
 class ClosureManager
 {
 public:
+    enum Action_t
+    {
+        CALIBRATE_ACTION   = 0,
+        MOVE_TO_ACTION     = 1,
+        STOP_MOTION_ACTION = 2,
+        STOP_CALIBRATE_ACTION = 3,
+
+        INVALID_ACTION     = 4
```

Comment:
No need to specify every value within the enum. Only the first one should be enough
```suggestion
        enum Action_t
        {
              CALIBRATE_ACTION = 0,
              MOVE_TO_ACTION,
              STOP_MOTION_ACTION,
              STOP_CALIBRATE_ACTION,

              INVALID_ACTION
        }
```